mw-webapp/src/App.tsx

```diff
@@ -1,14 +1,36 @@
+import {useEffect, useState} from "react";
+import {useNavigate} from "react-router-dom";
+import {getAuth, User} from "firebase/auth";
 import {Header} from "src/component/header/Header";
 import {Router} from "src/router/Router";
+import {handleUserAuthState} from "src/service/auth/handleUserAuthState";
 
 /**
  * App
  */
 export const App = () => {
+  const [user, setUser] = useState<User | null>(null);
+  const authObject = getAuth();
+  const navigate = useNavigate();
+  useEffect(() => {
+    handleUserAuthState(setUser);
+  }, []);
+
+  useEffect(() => {
+    if (authObject.currentUser) {
+      localStorage.setItem("auth", "true");
```

Comment:
magic values bring bugs, please define a variable and reuse it. Also, try to find the best naming to clarify the meaning of this variable. Because in month record with key auth will be looking strange, agree?

Reply:
done

Reply:
done